roles/semaphore_deployment/templates/deploy.j2

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
```

Comment:
```suggestion

set -e
```

This makes sure that the script fails if any of the commands fail. In this case it just makes sure that the script fails if ansible-playbook fails.